### PR TITLE
added MSTG-ID for iOS Chapter

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1139.

### CODE REVIEW

File: Document/0x06e-Testing-Cryptography.md [hunk 1]
```diff
@@ -35,14 +35,14 @@ As noted before: some wrapper-libraries exist for both in order to provide conve
...
+- **LibSodium**: Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, password hashing and more. It is a portable, cross-compilable, installable, packageable fork of NaCl, with a compatible API, and an extended API to improve usability even further. See [LibSodiums documentation](https://download.libsodium.org/doc/installation "LibSodium docs") for more details. There are some wrapper libraries, such as [Swift-sodium](https://github.com/jedisct1/swift-sodium "Swift-sodium"), [NAChloride](https://github.com/gabriel/NAChloride "NAChloride"), and [libsodium-ios](https://github.com/mochtu/libsodium-ios "libsodium ios").
+- **Tink**: A new cryptography library by Google. Google explains its reasoning behind the library [in its security blog](https://security.googleblog.com/2018/08/introducing-tink-cryptographic-software.html "Introducing Tink"). The sources can be found at [Tinks GitHub repository](https://github.com/google/tink "Tink at GitHub").
```
Comment (TheDauntless) [inline Document/0x06e-Testing-Cryptography.md]: *on* its blog

Comment (sushi2k) [reply Document/0x06e-Testing-Cryptography.md]: done :-)

File: Document/0x06a-Platform-Overview.md [hunk 2]
```diff
@@ -158,16 +158,16 @@ The following APIs [require user permission](https://www.apple.com/business/docs
...
-- Securely communicate with backend servers or is susceptible to man-in-the-middle (MitM) attacks between the server and the mobile application. See:
+- Securely communicate with backend servers or is susceptible to man-in-the-middle (MITM) attacks between the server and the mobile application, see also:
   - [Testing Network Communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication "Testing Network Communication");
   - [iOS Network APIs](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06g-Testing-Network-Communication.md#ios-network-apis "iOS Network APIs") .
-- Securely stores all local data, or loads untrusted data from storage. See:
+- Securely stores all local data, or loads untrusted data from storage, see also:
   - [Data Storage on iOS](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06d-Testing-Data-Storage.md#data-storage-on-ios "Data Storage on iOS").
```
Comment (cpholguera) [inline Document/0x06a-Platform-Overview.md]:
```suggestion
  - [Data Storage on iOS](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06d-Testing-Data-Storage.md#data-storage-on-ios "Data Storage on iOS")
```

File: Document/0x06a-Platform-Overview.md [hunk 3]
```diff
@@ -158,16 +158,16 @@ The following APIs [require user permission](https://www.apple.com/business/docs
...
+- Validate the content loaded inside a WebView, see also:
+  - [Testing iOS WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews");
   - [Determining Whether Native Methods Are Exposed Through WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#determining-whether-native-methods-are-exposed-through-webviews "Determining Whether Native Methods Are Exposed Through WebViews")
-- Securely communicate with backend servers or is susceptible to man-in-the-middle (MitM) attacks between the server and the mobile application. See:
+- Securely communicate with backend servers or is susceptible to man-in-the-middle (MITM) attacks between the server and the mobile application, see also:
   - [Testing Network Communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication "Testing Network Communication");
   - [iOS Network APIs](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06g-Testing-Network-Communication.md#ios-network-apis "iOS Network APIs") .
```
Comment (cpholguera) [inline Document/0x06a-Platform-Overview.md]:
```suggestion
  - [iOS Network APIs](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06g-Testing-Network-Communication.md#ios-network-apis "iOS Network APIs")
```

File: Document/0x06a-Platform-Overview.md [hunk 4]
```diff
@@ -158,16 +158,16 @@ The following APIs [require user permission](https://www.apple.com/business/docs
...
-  - [Testing iOS webviews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews");
+- Validate the content loaded inside a WebView, see also:
+  - [Testing iOS WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews");
   - [Determining Whether Native Methods Are Exposed Through WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#determining-whether-native-methods-are-exposed-through-webviews "Determining Whether Native Methods Are Exposed Through WebViews")
-- Securely communicate with backend servers or is susceptible to man-in-the-middle (MitM) attacks between the server and the mobile application. See:
+- Securely communicate with backend servers or is susceptible to man-in-the-middle (MITM) attacks between the server and the mobile application, see also:
   - [Testing Network Communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication "Testing Network Communication");
```
Comment (cpholguera) [inline Document/0x06a-Platform-Overview.md]:
```suggestion
  - [Testing Network Communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication "Testing Network Communication")
```

File: Document/0x06a-Platform-Overview.md [hunk 5]
```diff
@@ -158,16 +158,16 @@ The following APIs [require user permission](https://www.apple.com/business/docs
...
 The iOS application attack surface consists of all components of the application, including the supportive material necessary to release the app and to support its functioning. The iOS application may be vulnerable to attack if it does not:
 
-- Validate all input by means of IPC communication or URL-schemes. See
+- Validate all input by means of IPC communication or URL-schemes, see also:
   - [Testing Custom URL Schemes](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-custom-url-schemes "Testing Custom URL Schemes").
 - Validate all input by the user in input fields.
-- Validate the content loaded inside a webview. See:
-  - [Testing iOS webviews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews");
+- Validate the content loaded inside a WebView, see also:
+  - [Testing iOS WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews");
```
Comment (cpholguera) [inline Document/0x06a-Platform-Overview.md]:
```suggestion
  - [Testing iOS WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews")
```

File: Document/0x06a-Platform-Overview.md [hunk 6]
```diff
@@ -158,16 +158,16 @@ The following APIs [require user permission](https://www.apple.com/business/docs
 
 The iOS application attack surface consists of all components of the application, including the supportive material necessary to release the app and to support its functioning. The iOS application may be vulnerable to attack if it does not:
 
-- Validate all input by means of IPC communication or URL-schemes. See
+- Validate all input by means of IPC communication or URL-schemes, see also:
   - [Testing Custom URL Schemes](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-custom-url-schemes "Testing Custom URL Schemes").
```
Comment (cpholguera) [inline Document/0x06a-Platform-Overview.md]:
```suggestion
  - [Testing Custom URL Schemes](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-custom-url-schemes "Testing Custom URL Schemes")
```

File: Document/0x06e-Testing-Cryptography.md [hunk 7]
```diff
@@ -179,7 +143,7 @@ Even when the sync of CoreData or Realm is protected by using `NSFileProtectionC
...
+The last three options are to use hardcoded encryption keys in the source code, having a predictable key derivation function based on stable attributes, and storing generated keys in places that are shared with other applications. Obviously, hardcoded encryption keys are not the way to go. This means every instance of the application uses the same encryption key. An attacker needs only to do the work once, to extract the key from the source code - whether stored natively or in Objective-C/Swift. Consequently, he can decrypt any other data that he can obtain which was encrypted by the application.
```
Comment (cpholguera) [inline Document/0x06e-Testing-Cryptography.md]: source code - whether -> dash or comma?

Comment (sushi2k) [reply Document/0x06e-Testing-Cryptography.md]: fixed

**Pull request merged**